### DISCUSSION TITLE
What's the difference between appwrite and supabase?

### DISCUSSION THREAD

**Discussion by NeroBlackstone** [appwrite/appwrite#1447] [Q&A] [answered]
[Supabase](https://github.com/supabase/supabase)
I found another open source BaaS platform today, what's the difference and how do I choice?

**Comment by christyjacob4** [comment, marked as answer]
It depends a lot on your personal preference. 
* Appwrite can be **self hosted extremely easily** and has support for **multi-tenancy** out of the box. Meaning a single instance of Appwrite supports unlimited accounts and projects. We have pre built images which you can spin up with just a single click on DO.
* Appwrite was designed to work along side your current stack and not replace it. So it plays really well with your current backend. 
* Appwrite's developer experience is built around simplicity. You will notice that throughout our documentation. All the SDK calls and methods are extremely predictable making it extremely easy to build your apps.
* Appwrite has support for cloud functions out of the box with over 15 runtimes! So you can easily implement any custom logic in here :+1: 
* We have SDKs for most of the popular languages and platforms including flutter, android, kotlin, python, php, javascript and more.
* Appwrite also has support for over 20 oauth providers giving you complete flexibility!  
* Appwrite has a completely stateless architecture. So scaling is limited only by the number of instances you can spin up! 
* Appwrite also has an amazing and thriving developer community where you can come and hangout :smile:  => appwrite.io/discord 

I can go on, but you get the point :slightly_smiling_face:

**Reply by NeroBlackstone** [reply]
This is a really wonderful and detailed answer!
Appwrite is a great project, I like it very much!
Thank you for your answer!

**Reply by christyjacob4** [reply]
Glad I could help @NeroBlackstone  :smile:

**Reply by mariansam** [reply]
While all of this is true, it's definitely written in favor of AppWrite. You can find Supabase's CEO's answer over at [Reddit](https://www.reddit.com/r/programming/comments/qpdgk3/open_source_firebase_alternative_supabase_raises/hjsy7yp/?context=3) or [Hacker News](https://news.ycombinator.com/item?id=29010484).
